src/TTabBar.cpp

```diff
@@ -147,15 +147,15 @@ QString TTabBar::tabName(const int index) const
     return result;
 }
 
-int TTabBar::tabIndex(const QString& name) const
+int TTabBar::tabIndex(const QString& profileName) const
```

Comment:
:frowning: Given that this class is actually a bit of a generic handler - i.e. someone else could use it for their own purposes - `profileName` would be meaningless in such other contexts,  whereas `name` relates to the use of `tabName` vs `tabIndex` in other parts of this class's code.

Reply:
We're using it in the context of Mudlet - so it's appropriate to be context-specific to reduce confusion. We're not a library...

Reply:
Well then - are you going to rename all the other bits that refer to *name*?

Reply:
Good call on consistency - added https://github.com/Mudlet/Mudlet/pull/4803